shared-components/footer.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<header>
+    <meta charset="utf-8">
+    <meta name="viewport">
+    <title>Header</title>
+    <link rel="stylesheet" href="./footer-styles.css">
+</header>
+
+<body>
+    <footer>
+        <div class="logo">
+            <h2>Grind</h2>
+        </div>
+        <div class="links">
+            <div class="footer-links-left">
+                <div class="link one">
+                    <h5>Shop</h5>
+                    <div class="options">
+                        <p>Coffee Cups</p>
+                        <p>Tea Cups</p>
+                        <p>Plates</p>
+                        <p>Bowls</p>
+                    </div>
+                </div>
+                <div class="link two">
+                    <h5>Info</h5>
+                    <div class="options">
+                        <p>Contact Us</p>
+                        <p>Privacy</p>
+                        <p>Shipping & Returns</p>
+                        <p>Payments</p>
+                    </div>
+                </div>
+                <div class="link three">
```

Comment:
class name should not be `one two three`
use:
"link link-one" "link link-two".
Also `one` `two` `three` classes is not needed for styling. We can get rid of these